src/main/java/team/gif/robot/Robot.java

```diff
@@ -65,6 +67,7 @@ public void robotInit() {
         elevator = new Elevator();
         collector = new Collector();
         collectorPneumatics = new CollectorPneumatics();
+        pigeon = new Pigeon();
```

Comment:
We should change the pigeon class to include the CAN ID as an argument. I think it is hardcoded in the pigeon class to use RobotMap.